fhir-path/src/main/java/com/ibm/fhir/path/function/FHIRPathAbstractTermFunction.java

```diff
@@ -1,11 +1,12 @@
 /*
- * (C) Copyright IBM Corp. 2020
+ * (C) Copyright IBM Corp. 2021
```

Comment:
I believe the convention is to add the new year to the original year
```suggestion
 * (C) Copyright IBM Corp. 2020, 2021
```

Reply:
Good catch. Will update

Reply:
Updated copyright header